RubeesEat/Pages/SplitBill.cshtml

```diff
@@ -1,28 +1,43 @@
 @page
 @inject IPersonRepository PersonRepository
 
-Split bill:
-<form id="splitBillForm" method="post" action="/api/bills">
+<head>
+    <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"/>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" crossorigin="anonymous">
+    <link rel="stylesheet" href="/css/site.css" />
+</head>
 
-    <label for="billDescription">Description:</label>
-    <input id="billDescription" t-id="billDescription" type="text" name="billDescription" required="required"></input><br>
+<div class="container mt-3">
+    <h1 class="text-center">Split bill:</h1>
+    <form id="splitBillForm" method="post" action="/api/bills" class="mx-auto mt-4" style="max-width: 600px;">
```

Comment:
Please don't apply manual margins to elements. Usually, they already have margins that should work well together. In cases where that is not the case, the fix is usually to use the `row` class as a wrapper around the content. When using columns we should always use a row as wrapper. Please check if we do this everywhere.